RadixWallet/Features/AssetsFeature/Components/FungibleAssetList/Components/Details/FungibleTokenDetails+View.swift

```diff
@@ -28,7 +28,8 @@ extension FungibleTokenDetails.State {
 	var detailsHeader: DetailsContainerWithHeaderViewState {
 		.init(
 			title: resource.metadata.get(\.name, prefetched: ownedFungibleResource?.metadata).map { $0 ?? L10n.Account.PoolUnits.unknownPoolUnitName },
-			amount: ownedFungibleResource?.amount.formatted(),
+			amount: ownedFungibleResource?.amount.nominalAmount.formatted(),
+			currencyWorth: ownedFungibleResource?.amount.fiatWorth?.currencyFormatted(applyCustomFont: false),
```

Comment:
why not custom font?

Reply:
customFont applies only currently for TotalValue in HomeScreen and AccounDetails.